test/common/fixtures.mjs

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable node-core/require-common-first, node-core/required-modules */
```

Comment:
Add `common/fixtures.mjs` for fixtures path resolve utility in ES modules. Please comment here if there is a better way to do it

Reply:
You should just be able to import `common` to do this?

Reply:
Yeah, I've tried to find it at [common/index.mjs](https://github.com/nodejs/node/blob/master/test/common/index.mjs) at first. There seems no such method to do fixtures path resolve. Please let me know if I missed

Reply:
Agreed fixtures.mjs is necessary.